cmd/batch-expire.go

```diff
@@ -535,18 +536,29 @@ func (r *BatchJobExpire) Start(ctx context.Context, api ObjectLayer, job BatchJo
 		return err
 	}
 
-	ctx, cancel := context.WithCancel(ctx)
-	defer cancel()
+	ctx, cancelCancel := context.WithCancelCause(ctx)
```

Comment:
```suggestion
	ctx, cancelCause := context.WithCancelCause(ctx)
```
lets call it `cancelCause` like other place